index.Js

```diff
@@ -0,0 +1,25 @@
+//here we are going to define and declare object and inside the object I define two arrays
+const object = {
+  input:['Hello','How are you?','What is your favourite colour?'],
+  output:['Hi','Great!','I have so many favorites it hard to choose one']
+  };
+  console.log(object);
+//here we define function called reply and this function will let the chatpot to give me the proper anwers based on the input the we give
+  function reply(){
+    //I defince variable called question and I put inside this variable the value (input from user)
+    let question = document.getElementById('input').value;
+    //I used if statement to check the input and give the outpot
+    if(object.input.includes(question)){
```

Comment:
You've written a good check for valid input. You can also only use the [`indexOf` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) instead of using both `includes` and `indexOf`. `indexOf` returns -1 if the value is not found in the array so you can use this to check if the input is valid _and_ get the index of the input.